assets/js/app.js

```diff
@@ -22,6 +41,14 @@ $( document ).ready(function() {
                 cb(matches);
           };
     };
+    
+    var getMeta = $.ajax({
```

Comment:
Later on every input type is going to have a `datavalue` attribute. This function is going to open up a json file and put it into the typeahead.js function. I made the pull request so earliy, because Richmond wanted something to see.